.github/workflows/kind-e2e.yaml

```diff
@@ -178,6 +189,13 @@ jobs:
         # Be KinD to these tests.
         kubectl scale -nknative-serving deployment/chaosduck --replicas=0
 
+    - name: Install Alpha Components
+      if: contains(matrix.test-flags, '--enable-alpha')
+      run: |
+        export GO111MODULE=on
+        export GOFLAGS=-mod=vendor
```

Comment:
Could put in `env:` and save some text

Reply:
true, I copy pasta-ed this from the main 'Install Knative Serving' bit but I can fix it up in both places

Reply:
done!